### ISSUE TITLE
CommonJS issue for Nuxt SSR import

### ISSUE THREAD

**Issue by aonghas** [martijnversluis/ChordSheetJS#1414]
Hi there, when using this with Nuxt 3 (SSR) it works fine in client but when deployed I get the following error:

```
import ChordSheetJS, { templateHelpers } from 'chordsheetjs';
                       ^^^^^^^^^^^^^^^
SyntaxError: Named export 'templateHelpers' not found. The requested module 'chordsheetjs' is a CommonJS module, which may not support all module.exports as named exports.
CommonJS modules can always be imported via the default export, for example using:

import pkg from 'chordsheetjs';
const { templateHelpers } = pkg;
```

Any suggestions?

**Comment by aonghas** [comment]
I think it's the same issue in https://github.com/martijnversluis/ChordSheetJS/issues/1180

Any chance templateHelpers can be added to the default export too?

Thanks!

**Comment by aonghas** [comment]
I seem to have fixed this with the following config (in case it helps anyone else in future):

in nuxt.config:
```
export default defineNuxtConfig({

  build: {
    transpile: ["chordsheetjs"],
  },
})
```

When importing:

```
import * as ChordSheetJS from 'chordsheetjs';
const { templateHelpers } = ChordSheetJS;

```

**Comment by martijnversluis** [comment]
@aonghas Thanks for reporting this! I just released `10.6.0` which adds `templateHelpers` to the default export 👌
